Makefile

```diff
@@ -11,9 +11,25 @@ ISORT_FLAGS=--py 35 --extend-skip pynitrokey/nethsm/client
 # whitelist of directories for flake8
 FLAKE8_DIRS=pynitrokey/nethsm pynitrokey/cli/nk3 pynitrokey/nk3
 
+.PHONY: init-fedora37
+init-fedora37:
+	sudo dnf install -y swig pcsc-lite-devel
+	$(MAKE) init
+
 # setup development environment
 init: update-venv
 
+ARGS=
+.PHONY: run rune
+run:
+	./venv/bin/nitropy $(ARGS)
+
+rune:
+	podman run --privileged --rm -it --entrypoint /bin/bash pynitrokey
```

Comment:
1. I prefer using a variable for the executable so that this works both with docker and podman.  In `opcard`, we call that variable [`$DOCKER`](https://github.com/Nitrokey/opcard-rs/blob/1e5b838e1cc95d92886e71c6587db7f80e1d2366/ci/Makefile#L7) and default to `docker`.  It would make sense to do this consistently within our projects so that developers can easily select the implementation they prefer in their profile.
2. Can you add a short mention of this target to the developer guide with a link to the relevant upstream docs?

Reply:
1. Good idea
2. Sure. I will maybe extract this to separate Makefile, as a less official way (since Earthly is not used in the team widely), and describe it in docs.